ftl/php/benchmark/main.py

```diff
@@ -12,92 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import subprocess
 import argparse
-import datetime
-import time
-import os
-import logging
-from google.cloud import bigquery
+from ftl.benchmark import args
+from ftl.benchmark import benchmark
+
 
 DATASET_NAME = 'ftl_benchmark'
```

Comment:
The convention used in other code for private constants is to prefix with '_'.  It is the 'pythonic' way to say a variable is private.  These should probably be _DATASET_NAME, etc.

Reply:
thanks!